wsagent/che-core-api-git/src/main/java/org/eclipse/che/api/git/GitChangesDetector.java

```diff
@@ -192,9 +194,9 @@ public void stopWatcher() {
                     .withEditedRegions(gitConnection.getEditedRegions(itemPath)))
             .sendAndSkipResult();
       } catch (NotFoundException | ServerException e) {
-        String errorMessage = e.getMessage();
-        if (!("Not a git repository".equals(errorMessage))) {
-          LOG.error(errorMessage);
+        if (!(e instanceof NotAGitRepositoryException)
```

Comment:
```
} catch (GitCommitInProgressException  | NotAGitRepositoryException e) {
  LOG.debug()
} catch (NotFoundException | ServerException e) {
```

Reply:
`GitCommitInProgressException` and `NotAGitRepositoryException` have already been caught by `ServerException`

Reply:
done